metas.groovy

```diff
@@ -0,0 +1,137 @@
+//Adapted from module.groovy

```

Comment:
For some reason it looks like you still have bad line endings. After I merged this locally the whole file immediately lit up as changed as the line endings were changed. Maybe look up a quick how to for figuring out how to check your Git's line ending related config and how to test it.

Reply:
But this time I used Atom !!! idk Whats happening...